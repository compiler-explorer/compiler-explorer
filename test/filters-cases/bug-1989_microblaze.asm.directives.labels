issue1989:
        addk    r6,r5,r6
        cmp     r18,r6,r5
        bltid   r18,$L2
        addik   r3,r0,1       # 0x1
        addk    r3,r0,r0
$L2:
        rtsd    r15,8

        andi    r3,r3,1 #and1
