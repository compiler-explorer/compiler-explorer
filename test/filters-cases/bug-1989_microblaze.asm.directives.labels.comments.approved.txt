{
  "asm": [
    {
      "labels": [],
      "source": null,
      "text": "issue1989:"
    },
    {
      "labels": [],
      "source": null,
      "text": "        addk    r6,r5,r6"
    },
    {
      "labels": [],
      "source": null,
      "text": "        cmp     r18,r6,r5"
    },
    {
      "labels": [
        {
          "name": "$L2",
          "range": {
            "endCol": 24,
            "startCol": 21
          }
        }
      ],
      "source": null,
      "text": "        bltid   r18,$L2"
    },
    {
      "labels": [],
      "source": null,
      "text": "        addik   r3,r0,1       # 0x1"
    },
    {
      "labels": [],
      "source": null,
      "text": "        addk    r3,r0,r0"
    },
    {
      "labels": [],
      "source": null,
      "text": "$L2:"
    },
    {
      "labels": [],
      "source": null,
      "text": "        rtsd    r15,8"
    },
    {
      "labels": [],
      "source": null,
      "text": ""
    },
    {
      "labels": [],
      "source": null,
      "text": "        andi    r3,r3,1 #and1"
    }
  ],
  "labelDefinitions": {
    "$L2": 7,
    "issue1989": 1
  }
}
